source/extensions/grpc_credentials/aws_iam/BUILD

```diff
@@ -4,17 +4,18 @@ licenses(["notice"])  # Apache 2
 
 load(
     "//bazel:envoy_build_system.bzl",
-    "envoy_cc_library",
+    "envoy_cc_extension",
     "envoy_package",
 )
 
 envoy_package()
 
-envoy_cc_library(
+envoy_cc_extension(
     name = "config",
     srcs = ["config.cc"],
     hdrs = ["config.h"],
     external_deps = ["grpc"],
+    security_posture = "unknown",
```

Comment:
cc @lavignes

Reply:
I believe this should be "requires_trusted_downstream_and_upstream". Technically the intended downstream is Envoy right now. But this could be used to intercept any grpc traffic.